Thus patch makes busybox 1.19 compilable on hurd again.
Hurd does not define CBAUD constant in termios.h, but
new getty.c (rewritten in busybox 1.19) uses that constant
unconditionally.

Signed-Off-By: Michael Tokarev <mjt@tls.msk.ru>

Index: debian/loginutils/getty.c
===================================================================
--- debian.orig/loginutils/getty.c	2011-10-22 16:05:00.455173199 +0400
+++ debian/loginutils/getty.c	2011-10-30 20:51:29.386553998 +0400
@@ -271,7 +271,9 @@
 #ifdef CMSPAR
 		| CMSPAR  /* mark or space parity */
 #endif
+#ifdef CBAUD
 		| CBAUD   /* (output) baud rate */
+#endif
 #ifdef CBAUDEX
 		| CBAUDEX /* (output) baud rate */
 #endif
